src/builder.rs

```diff
@@ -94,3 +98,135 @@ impl TransactionBuilder {
         Ok((ReissueTransaction { inputs, outputs }, output_owners))
     }
 }
+
+/// A Builder for aggregating ReissueShare (Mint::reissue() results)
+/// from multiple mint nodes and combining signatures to
+/// generate the final Dbc outputs.
+#[derive(Default)]
+pub struct DbcBuilder {
+    pub reissue_transaction: Option<ReissueTransaction>,
+    pub reissue_shares: Vec<ReissueShare>,
+}
+
+impl DbcBuilder {
+    /// Create a new DbcBuilder from a ReissueTransaction
+    pub fn new(reissue_transaction: ReissueTransaction) -> Self {
+        Self {
+            reissue_transaction: Some(reissue_transaction),
+            reissue_shares: Default::default(),
+        }
+    }
+
+    /// Add a ReissueShare from Mint::reissue()
+    pub fn add_reissue_share(mut self, reissue_share: ReissueShare) -> Self {
+        self.reissue_shares.push(reissue_share);
+        self
+    }
+
+    /// Set the ReissueTransaction
+    pub fn set_reissue_transaction(mut self, reissue_transaction: ReissueTransaction) -> Self {
+        self.reissue_transaction = Some(reissue_transaction);
+        self
+    }
+
+    /// Build the output DBCs
+    ///
+    /// Note that the result Vec may be empty if the ReissueTransaction
+    /// has not been set or no ReissueShare has been added.
+    pub fn build(self) -> Result<Vec<Dbc>> {
+        if self.reissue_shares.is_empty() {
+            return Ok(vec![]);
```

Comment:
I think this should be an error, the reissue transaction will likely specify output DBC's so it seems odd to me that an Ok result would not match the number of output DBC's